docs/userguides/testing.md

```diff
@@ -392,10 +392,51 @@ def contract(owner, project):
     return owner.deploy(project.MyContract)
 
 def test_unauthorized_withdraw(contract, hacker):
-    with ape.reverts(contract.Unauthorized, addr=hacker.address):
+    with ape.reverts(contract.Unauthorized, unauth_address=hacker.address):
         contract.withdraw(sender=hacker)
 ```
 
+You can also use the ABI of the error if you don't have access to an instance:
```

Comment:
Is this sentence correct?

Reply:
Yes? I think so? is this a trick question?

Reply:
Using the ABI is the `projet.MyContract.MyError` is the way to do if you dont have access to an instance yet, such as during deploys or if you just dont care or if you dont have access to it

Reply:
I have an idea.... standby!

Reply:
I meant the way it was phrased seemed weird and didn't make sense to me

Reply:
OK made the statement not true and changed the statement and made it so custom error types are accessible from the container, and not just the ABI alone, this may help with future mocking features we want to consider, similar to what we did for `MockContractLog` perhaps

Reply:
oh well i just did a large-ish refactor because of it haha because i thought you were saying something else. I think maybe was ok though.

Reply:
The sentence is now

```
You can also use custom error types from the contract container (from `ape.project` or the `project` fixture):
```

let me know if it is still unclear!